src/index.html

```diff
@@ -12,13 +12,16 @@
     <meta name="format-detection" content="telephone=no" />
     <meta name="msapplication-tap-highlight" content="no" />
     <!-- Ikke endre direkte. Sjekk README.md for å se hvordan man oppdaterer Content-Security-Policy -->
-    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' https://plausible.io https://offlinemap.blob.core.windows.net https://nveb2c01prod.b2clogin.com https://nveb2c01test.b2clogin.com https://nveb2c01staging.b2clogin.com https://api.regobs.no https://test-api.regobs.no https://demo-api.regobs.no http://localhost:5130/ http://localhost:40001/ https://api01.nve.no https://ws.geonorge.no/stedsnavn https://secure.geonames.org https://www.iskart.no https://sentry.io https://ws.geonorge.no https://www.yr.no data:; frame-src https://plot.regobs.no https://test-plot.regobs.no https://demo-plot.regobs.no; form-action 'self'; object-src 'none'; font-src 'self'; style-src-elem 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' https://plausible.io; worker-src 'self' blob:; default-src 'self' data: gap: cdvfile: blob:; img-src * filesystem: app-file: cdvfile: data: blob:; media-src * blob:" />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="connect-src 'self' https://plausible.io https://offlinemap.blob.core.windows.net https://nveb2c01prod.b2clogin.com https://nveb2c01test.b2clogin.com https://nveb2c01staging.b2clogin.com https://api.regobs.no https://test-api.regobs.no https://demo-api.regobs.no http://localhost:5130/ http://localhost:40001/ https://api01.nve.no https://ws.geonorge.no/stedsnavn https://secure.geonames.org https://www.iskart.no https://sentry.io https://ws.geonorge.no https://www.yr.no data:; frame-src https://plot.regobs.no https://test-plot.regobs.no https://demo-plot.regobs.no; form-action 'self'; object-src 'none'; font-src 'self'; style-src-elem 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' https://plausible.io; worker-src 'self' blob:; default-src 'self' data: gap: cdvfile: blob:; img-src * filesystem: app-file: cdvfile: data: blob:; media-src * blob:"
+    />
 
     <link rel="apple-touch-icon" sizes="180x180" href="assets/icon/apple-touch-icon.png" />
     <link rel="icon" type="image/png" href="assets/icon/favicon.ico" />
 
     <!-- add to homescreen for ios -->
-    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="mobile-web-app-capable" content="yes" />
```

Comment:
Det er ikke noe spesiell grunn til at denne er endra? Hvis ikke tror jeg vi bør reversere den, kan ikke se at `mobile-web-app-capable` er gyldig, mens `apple-mobile-web-app-capable` fortsatt har en funksjon på iOS.

```suggestion
    <meta name="apple-mobile-web-app-capable" content="yes" />
```

Reply:
Ja, jeg fikk en advarsel i konsollet om at den funksjonen var utdatert. Det var derfor jeg endret den.

Reply:
Men jeg kan fjerne den egen så kan vi heller ta det seinere evt.